tests/unit/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * checkboxradio_core.js
+ */
+
+
+(function($) {
+
+module("Checkboxradio: core");
+test("Checkbox", function() {
+	expect( 4 );
+	var input = $("#check"),
+		label = $("label[for=check]");
+	ok( input.is( ":visible" ) );
+	ok( !label.hasClass(".ui-button)") );
```

Comment:
I consider this a good example for my earlier proposal to explicitly test the class attribute.